FogCreek/Glitch-Community
# Long city: Exploring Collections

## Links
* Remix link: https://long-city.glitch.me/@glitch/glitch-team-faves/play
* Issue-tracker link: Covers a lot of different tickets, but here's the board with all the related epic tickets: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/board?epics=192398789_428&repos=117580951,192398789
* Specs/doc links: https://docs.google.com/document/d/1VMTVJE-ZEGA2dIqBknaM25pkODMyfYAEK5FEVPWkPvI/edit#heading=h.3im6lap0txtw

## GIF/Screenshots:
Screencapture: https://recordit.co/ZoSppzVn6v

## Changes:
* Adds a `Play` button to collections
* When you click that button it launches a new play view of collections where you can look at a project embed of each project one at a time.
* Doing this will also change the url so that a particular point in a collection is sharable as well. It'll add the `play/:projectId` to the url, if no projectId is passed in it will default to the first project in a collection

## How To Test:
There is no dev toggle, I'm just putting this all in one big pr, since it seemed relatively self contained to one page. Just go to any collection and check it out! Some particular edge cases to think about:
- how does it look when there are no projects in a collection
- how does it look when there's 1 project in a collection
- how is adding/removing notes
- does reordering a collection mess with anything?
- does the browser's back/forward work as expected
- can you add something to my stuff
- ?? probably other things! try to break it plz :)

## Feedback I'm looking for:
Anything!

## Things left to do before deploying:
- [x] add metrics
- [x] browser testing
- [x] a11y testing
- [x] not sure if we want to do a formal qa session for this mini project or not, will ask the pod.
